src/tests/animeSlice.test.js

```diff
@@ -0,0 +1,106 @@
+import animeReducer, { filtergenre, searchanime } from '../redux/animelist/animeSlice';
+
+describe('test animelist', () => {
+  it('Type of filter genre', () => {
+    const action = filtergenre(3);
+    const ecpectedAction = {
+      payload: 3,
+      type: 'animelist/filtergenre',
+    };
+
+    expect(action).toEqual(ecpectedAction);
+  });
+
+  it('Type of searchanime anime', () => {
```

Comment:
**You have done a great job!**

- [ ] The only issue is with your tests. Almost all tests are passing except this one searchanime() looks like you are not passing any value for it. Please fix the issue and resubmit your code to get approved. Thank you!

This is the screenshot of the failing test:
<img width="493" alt="Screenshot 2023-08-04 at 15 52 15" src="https://github.com/vikas924/react-capstone-project/assets/69011963/712bcfc5-50a5-42a1-98ec-9f3142e67192">